ui/apollo.config.js

```
@@ -1,8 +1,19 @@
+// apollo.config.js
+const { readdirSync, readFileSync, writeFileSync } = require('fs');
+
+const schemasFolder = __dirname + '/../api/graphql/resolvers';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)